### ISSUE TITLE
Test suite compilation failure

### ISSUE THREAD

**Issue by snoyberg** [axellang/axel#22]
The Stackage build failed to compile the test suite, I'm going to temporarily remove the package in the meanwhile:

```
[10 of 18] Compiling Axel.Test.Haskell.StackSpec ( test/Axel/Test/Haskell/StackSpec.hs, dist/build/axel-test/axel-test-tmp/Axel/Test/Haskell/StackSpec.o )

test/Axel/Test/Haskell/StackSpec.hs:45:20: error:
    • No instance for (MonadIO Identity)
        arising from a use of ‘buildStackProject’
    • In the expression: buildStackProject "project/path"
      In an equation for ‘action’:
          action = buildStackProject "project/path"
      In the second argument of ‘($)’, namely
        ‘do let action = buildStackProject "project/path"
            let origFSState = mkFSState ...
            let origProcState = mkProcessState ... ...
            let expectation result (procState, fsState) = ...
            ....’
   |
45 |       let action = Stack.buildStackProject "project/path"
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by jgrosso** [comment]
@snoyberg Just published v0.0.6 to Hackage, which should no longer cause this issue.

**Comment by snoyberg** [comment]
I've pushed a commit to reenable axel.
